neo/Plugins/Plugin.cs

```diff
@@ -110,6 +118,14 @@ internal static void LoadPlugins(NeoSystem system)
             }
         }
 
+        internal static void NotifyPluginsLoadedAfterSystemConstructed()
+        {
+            foreach (var plugin in Plugins)
+                plugin.OnNeoSystemInitialized();
```

Comment:
I needed to add this also, because some plugins like ApplicationLogs need to do some setup after the `Blockchain` actor is created and before `OnPluginsLoaded` where the import blocks will start importing blocks.

Reply:
This looks strange to me, because both foreach are sequential and called one after the other.

Reply:
I reverted adding this hook. Not needed by anything after changing ApplicationLogs to subscribe to the EventStream earlier.